apps/studio/src/server/modules/auth/email/email.router.ts

```diff
@@ -19,17 +19,19 @@ export const emailSessionRouter = router({
   login: publicProcedure
     .input(emailSignInSchema)
     .mutation(async ({ ctx, input: { email } }) => {
-      // check if whitelisted email on Growthbook
-      const defaultWhitelist: string[] = []
-      const whitelistedUsers = ctx.gb.getFeatureValue("whitelisted_users", {
-        whitelist: defaultWhitelist,
-      })
-
-      if (!whitelistedUsers.whitelist.includes(email)) {
-        throw new TRPCError({
-          code: "UNAUTHORIZED",
-          message: "Unauthorized. Contact Isomer support.",
+      if (env.NODE_ENV === "production") {
```

Comment:
our NODE_ENV as set in the task_definition is always `production` when deployed - be it on staging or production infra

Reply:
This whitelist check is a temp measure till we get permissions model up

Reply:
just to be clear on this, we should always fix `NODE_ENV` to `production` and have htis conditional changed to maybe `ENV` right?

Reply:
yea that's right for dev clarity we should have separation of staging vs local vs prod. But for this purpose, we want to do this check on both stg + prod in which this conditional check for NODE_ENV works